framework/src/timeintegrators/CrankNicolson.C

```diff
@@ -39,17 +40,6 @@ CrankNicolson::computeTimeDerivatives()
 void
 CrankNicolson::init()
 {
-  // make sure that time derivative contribution is zero in the first pre-solve step
-  _u_dot.zero();
```

Comment:
I can add `_u_dot.zero()` and `_du_dot_du = 0` back if you think we should.

Reply:
👍